README.md

```diff
@@ -263,8 +262,9 @@ Setting any of the following options will enable an SSL connection, but only if
 your MySQL client library and server have been compiled with SSL support.
 MySQL client library defaults will be used for any parameters that are left out
 or set to nil. Relative paths are allowed, and may be required by managed
-hosting providers such as Heroku. Set `:sslverify => true` to require that the
-server presents a valid certificate.
+hosting providers such as Heroku. For MySQL versions > `5.7.11` do not use `:sslverify` 
```

Comment:
I suggest to rephrase this in the positive, how about:
> For MySQL versions 5.7.11 and higher, use `:ssl_mode` to set a preferred or required SSL connections and certificate validation level. For earlier versions of MySQL, use the `:sslverify` boolean.

Reply:
Off the top of my head, I don't recall the specific behavior if the user set `:sslverify` when connecting to MySQL 8. Based on the deprecation and now removal of this option, I think the mysql2 gem should enforce migrating users to :ssl_mode by way of warnings and failing the connection.

Reply:
Alright, changing it to sound more positive makes sense. Regarding the behaviour, I can open up an issue to track it.